indigo_app/views/places.py

```diff
@@ -765,6 +766,36 @@ def get_success_url(self):
         return reverse('place_settings', kwargs={'place': self.kwargs['place']})
 
 
+class PlaceUsersView(PlaceViewBase, FormView):
+    template_name = 'place/users.html'
+    tab = 'place_users'
+    form_class = PlaceUsersForm
+
+    def get_initial(self):
+        initial = super().get_initial()
+        initial["users"] = [u.id for u in User.objects.filter(Q(editor__permitted_countries=self.place) | Q(is_superuser=True))]
+        return initial
+
+    def form_valid(self, form):
+        users = User.objects.all().exclude(is_superuser=True)
+        editor_badge = badges.registry.get('editor')
+        country_badge = badges.registry.get(f"country-{self.country.code.lower()}")
+        for user in users:
+            if str(user.id) in form.cleaned_data['users']:
+                editor_badge.possibly_award(user=user)
+                country_badge.possibly_award(user=user)
+
+            else:
+                editor_badge.unaward(user=user)
```

Comment:
we don't want to unaward editor badges — is that what this is doing?

Reply:
hmm, we don't want to award editor badges either — that will still get done on a per-user basis

Reply:
I think you can ignore the editor badge completely here

Reply:
yes, so removing the country badge would be sufficient?